Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -43,6 +44,69 @@ YUI.add('ez-locationviewviewservice', function (Y) {
         },
 
         /**
+         * copyAction event handler, launch the universal discovery widget
+         * to choose a location to copy the content
+         *
+         * @method _copySelectLocation
+         * @protected
+         * @param {EventFacade} e
+         */
+        _copySelectLocation: function (e) {
+            this.fire('contentDiscover', {
+                config: {
+                    title: "Select the location you want to copy your content into",
+                    contentDiscoveredHandler: Y.bind(this._copyContent, this),
+                },
+            });
+        },
+
+        /**
+         * Copy the content to the selected location
+         *
+         * @method _copyContent
+         * @protected
+         * @param {EventFacade} e
+         */
+        _copyContent: function (e) {
+            var app = this.get('app'),
+                parentLocationId = e.selection.location.get('id'),
+                locationId = this.get('location').get('id'),
+                that = this,
+                notificationIdentifier =  'copy-notification-' + parentLocationId + '-' + locationId;
+
+            this._fireNotification('The copy operation of your content has begun', notificationIdentifier, 'started', 5);
+            this.get('content').copy({api: this.get('capi')}, parentLocationId, function (error, response) {
+                if (error) {
+                    that._fireNotification('An error occured while copying your content', notificationIdentifier, 'error', 5);
+                    return;
+                }
+                that._fireNotification('Your content has been successfully copied', notificationIdentifier, 'done', 5);
+                app.navigateTo('viewLocation', {id: response.getHeader('location')});
+            });
+        },
+
+        /**
+         * Fire 'notify' event
+         *
+         * @method _fireNotify
+         * @protected
+         * @param {String} text the text shown durring the notification
+         * @param {String} identifier the identifier of the notification
+         * @param {String} state the state of the notification
+         * @param {Integer} timeout the number of second, the notification will be shown
+         */
+        _fireNotification: function (text, identifier, state, timeout) {
```

Comment:
As @dpobel asked for it in another question - can name of this method be changed to `_notify`?